tembo-operator/src/stacks/mod.rs

```diff
@@ -22,6 +22,8 @@ lazy_static! {
         serde_yaml::from_str(include_str!("templates/vectordb.yaml")).expect("vectordb.yaml not found");
     pub static ref GEOSPATIAL: Stack =
         serde_yaml::from_str(include_str!("templates/gis.yaml")).expect("gis.yaml not found");
+    pub static ref MONGO_ADAPTER: Stack = serde_yaml::from_str(include_str!("templates/mongo_adapter.yaml"))
```

Comment:
is it too much trouble to name anything that's "Mongo" to "Ferret" for consistency and trademark concerns. Also I think it should just be called "FERRET_DB" or just "FERRET", not including "ADAPTER"

Reply:
The functionality provides users a Mongo compatible protocol with Postgres as the backend storage.